cxx-squid/src/test/java/org/sonar/cxx/preprocessor/SourceCodeProviderTest.java

```diff
@@ -118,6 +118,14 @@ public void getting_file_with_filename_and_cwd() {
   }
 
   @Test
+  public void getting_file_with_filename_and_cwd_no_extension() {
+    String cwd = new File("src/test/resources/codeprovider").getAbsolutePath();
+    String path = "source";
```

Comment:
@guwirth this is supposed to mimic a include directive, its not the real source file in filesystem